bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -73,14 +73,26 @@ impl Client {
 
         let (sliding_sync_reset_broadcast_tx, _) = broadcast::channel(1);
 
-        Client {
+        let client = Client {
             client,
             state: Arc::new(RwLock::new(state)),
             delegate: Arc::new(RwLock::new(None)),
             session_verification_controller,
             sliding_sync_proxy: Arc::new(RwLock::new(None)),
             sliding_sync_reset_broadcast_tx,
-        }
+        };
+
+        let mut client_api_error_subscription = client.client_api_error_broadcast().subscribe();
+        let client_clone = client.clone();
+        RUNTIME.spawn(async move {
+            loop {
+                if let Ok(error) = client_api_error_subscription.recv().await {
+                    client_clone.process_api_client_error(error)
+                }
```

Comment:
The potential `Err` cases here should be handled. `Err(RecvError::Closed)` should end the loop, `Err(RecvError::Lagged(_))` can probably be ignored.